### PR TITLE
{chem}[intel/2019b] LAMMPS v7Aug2019 w/ Python 3.7.4 w/ OCTP

### PR DESCRIPTION
**Pull request by boegel**
(created using `eb --new-pr`)

### CODE REVIEW

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node2621.swalot.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) CPU E5-2660 v3 @ 2.60GHz (haswell), Python 2.7.5
See https://gist.github.com/28bcfe9d24c5b9d2dd2f2f7d075fe560 for a full test report.

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3409.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/a047aa1ab26bebae8ef1929f92156cbb for a full test report.

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by casparvl** [comment]
Test report by @casparvl
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
software1.lisa.surfsara.nl - Linux debian 10.5, x86_64, Intel(R) Xeon(R) Silver 4110 CPU @ 2.10GHz, Python 2.7.16
See https://gist.github.com/e6f1ed2c9ddc5d78fc207472fd09d85f for a full test report.

**Comment by casparvl** [comment]
Test report by @casparvl
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
tcn900.bullx - Linux RHEL 7.7, x86_64, Intel(R) Xeon(R) CPU E5-2690 v3 @ 2.60GHz, Python 2.7.5
See https://gist.github.com/c2dee89ed00f55ac3782cee2a7a6ec1f for a full test report.

File: easybuild/easyconfigs/l/LAMMPS/LAMMPS-7Aug2019-intel-2019b-Python-3.7.4-kokkos-OCTP.eb [hunk 1]
```diff
@@ -0,0 +1,112 @@
+name = 'LAMMPS'
+version = '7Aug2019'
+local_python_versionsuffix = '-Python-%(pyver)s'
+versionsuffix = local_python_versionsuffix
+versionsuffix += '-kokkos-OCTP'
+
+homepage = 'https://lammps.sandia.gov/'
+description = """LAMMPS is a classical molecular dynamics code, and an acronym
```
Comment (casparvl) [inline easybuild/easyconfigs/l/LAMMPS/LAMMPS-7Aug2019-intel-2019b-Python-3.7.4-kokkos-OCTP.eb:8]: I think the description should also contain something on OCTP. I for one have no idea what it is, and what would make this installation different from a standard LAMMPS.

File: easybuild/easyconfigs/l/LAMMPS/LAMMPS-7Aug2019-intel-2019b-Python-3.7.4-kokkos-OCTP.eb [hunk 2]
```diff
@@ -0,0 +1,112 @@
...
+parallel using message-passing techniques and a spatial-decomposition of the
+simulation domain. The code is designed to be easy to modify or extend with new
+functionality.
+"""
+
+toolchain = {'name': 'intel', 'version': '2019b'}
+toolchainopts = {'openmp': True, 'usempi': True, 'cstd': 'c++11'}
+
+source_urls = [
+    # 'https://github.com/lammps/lammps/archive/'
```
Comment (casparvl) [inline easybuild/easyconfigs/l/LAMMPS/LAMMPS-7Aug2019-intel-2019b-Python-3.7.4-kokkos-OCTP.eb:23]: Is it really useful to leave this here? We also don't 'explain'  in a comment what `GITHUB_LOWER_SOURCE` is in other EC's, right?

**Comment by hajgato** [comment]
@boegel maybe you should also add LAMMPS EPYC patch (for kokkos) https://github.com/easybuilders/easybuild-easyconfigs/pull/11334

File: easybuild/easyconfigs/l/LAMMPS/LAMMPS-7Aug2019-intel-2019b-Python-3.7.4-kokkos-OCTP.eb (hunk 1 above)
Comment (boegel) [reply easybuild/easyconfigs/l/LAMMPS/LAMMPS-7Aug2019-intel-2019b-Python-3.7.4-kokkos-OCTP.eb:8]: done in 07d1e28

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
@boegelbot please test @ generoso

**Comment by boegel** [comment]
Test report by @boegel
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
node3404.kirlia.os - Linux centos linux 7.8.2003, x86_64, Intel(R) Xeon(R) Gold 6240 CPU @ 2.60GHz (cascadelake), Python 2.7.5
See https://gist.github.com/b5a33bc6a45f3416da94ddf331db0600 for a full test report.

**Comment by casparvl** [comment]
Test report by @casparvl
**FAILED**
Build succeeded for 0 out of 1 (1 easyconfigs in this PR)
software1.lisa.surfsara.nl - Linux debian 10.5, x86_64, Intel(R) Xeon(R) Silver 4110 CPU @ 2.10GHz, Python 2.7.16
See https://gist.github.com/e697f2e2758686fe2eb93a2458612e70 for a full test report.

**Comment by casparvl** [comment]
Test report by @casparvl
**SUCCESS**
Build succeeded for 1 out of 1 (1 easyconfigs in this PR)
software1.lisa.surfsara.nl - Linux debian 10.5, x86_64, Intel(R) Xeon(R) Silver 4110 CPU @ 2.10GHz, Python 2.7.16
See https://gist.github.com/076a7fbe8956a8f5ae7bbae57598b509 for a full test report.

**Review by casparvl — APPROVED** [review]

**Comment by casparvl** [comment]
LGTM, going in! Thx @boegel

**Pull request merged**